app/products/calls/alerts.ts

```diff
@@ -23,12 +23,18 @@ import {isSystemAdmin} from '@utils/user';
 import type {LimitRestrictedInfo} from '@calls/observers';
 import type {IntlShape} from 'react-intl';
 
-// Only allow one recording alert per call.
+// Only unlock when:
+// - Joining a new call.
+// - A new recording has started.
+// - Host has changed to current user.
 let recordingAlertLock = false;
```

Comment:
This should have been:
```suggestion
let recordingAlertLock = true;
```